src_assets/common/assets/web/config.html

```diff
@@ -561,6 +561,13 @@ <h2 class="accordion-header">
           <div class="form-text">{{ $t('config.origin_web_ui_allowed_desc') }}</div>
         </div>
 
+        <!-- Web UI Address -->
+        <div class="mb-3">
+          <label for="web_ui_address" class="form-label">{{ $t('config.web_ui_address') }}</label>
+          <input type="text" class="form-control" id="web_ui_address" placeholder="https://localhost:47990" v-model="config.web_ui_address" />
```

Comment:
Should I change the placeholder to a reverse proxy example? If yes, is `https://example.com:80/sunshine` appropriate here (and elsewhere)? I'm asking because the `External IP` entry is using an example IP range, so I assume users won't get confused that the placeholder is only an example and not the actual default when unset.